pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -72,52 +65,62 @@ buildDotnetModule rec {
     libX11
   ];
 
-  executables = [
-    nexusmods-app.meta.mainProgram
-  ];
+  executables = [ meta.mainProgram ];
```

Comment:
```suggestion
  executables = [ "NexusMods.App" ];
```

Reply:
Thanks for the feedback, but could you clarify the motivation?

When I discussed this earlier (https://github.com/NixOS/nixpkgs/pull/331355#discussion_r1699005403) it was thought that using `meta.mainProgram` via `rec` or `finalAttrs` would be fine.

Reply:
@SuperSandro2000 

You asked for the same thing in https://github.com/NixOS/nixpkgs/pull/331150#discussion_r1698368777, but never explained your reasoning.